misc/AdditionalSetupDuplicateUsers.ps1

```diff
@@ -0,0 +1,16 @@
+Write-Host "Finding users with duplicate AuthenticationEmail and fixing disabled duplicate users"
+
+$duplicateAadUserSets = Get-NAVServerUser -ServerInstance $ServerInstance | where { $_.AuthenticationEmail -ne '' } | group 'AuthenticationEmail' | Where { $_.Count -gt 1 } 
+
+foreach ($duplicateAadUserSet in $duplicateAadUserSets)
+{
+    $disabledDuplicateUsers = $duplicateAadUserSet.Group | Where-Object { $_.State -eq 'Disabled' }
+
+    foreach ($disabledDuplicateUser in $disabledDuplicateUsers)
+    {
+        Write-Host "Fixing AuthenticationEmail of duplicate user $($_.UserName)"
+
+        # moving to a non-existent email as removing doesn't work
+        Set-NAVServerUser -ServerInstance $ServerInstance -UserName $_.UserName -AuthenticationEmail "none@example.com"
```

Comment:
Doesn't this lead to duplicated "none@example.com"?

Reply:
yes it does but BC doesn't care. The reason behind this error was that when there are two users with the same AuthenticationEmail (AAD auth) BC sometimes tries to use the disabled one during the sign in. With changing the email the disabled duplicate is not selected anymore by BC